pyronear/datasets/openfire.py

```diff
@@ -42,13 +54,16 @@ class OpenFire(VisionDataset):
     seed = 42
 
     def __init__(self, root, train=True, transform=None, target_transform=None,
-                 download=False, threads=16, valid_pct=None):
+                 download=False, threads=16, valid_pct=0.2, max_files=-1):
         super(OpenFire, self).__init__(root, transform=transform,
                                     target_transform=target_transform)
         self.train = train  # training set or test set
-
-        if download:
-            self.download(threads, valid_pct)
+        
+        if not (self._check_exists(train=True) and self._check_exists(train=False)):
+          self.create_directories(max_files=max_files)
+          if download:
+              self.download(threads)
+          self.check_images(valid_pct=valid_pct)
```

Comment:
Spacing issue here, one tab should be equal to 4 spaces in your editor (rather than 2 here most likely)